opm/grid/common/ZoltanPartition.cpp

```diff
@@ -50,7 +50,7 @@ zoltanGraphPartitionGridOnRoot(const CpGrid& cpgrid,
     {
         OPM_THROW(std::runtime_error, "Could not initialize Zoltan!");
     }
-
+    Zoltan_Set_Param(zz, "IMBALANCE_TOL", "1.1");
```

Comment:
Out of curiosity what is the default?

Reply:
The default is "1.1". I sometimes test with lower imbalance tolerance, and forgot to remove this line. 

Finding the imbalance tolerance default in the Zoltan documentation is annoying, so leaving this line adds some valuable documentation.